book/src/dev/rfcs/xxxx-block-subsidy.md

```diff
@@ -6,32 +6,43 @@
 
 # Motivation
 [motivation]: #motivation
 
-All incoming blocks must be validated with the protocol rules, there is no way to avoid this logic to be present in any Zcash protocol compatible implementation such as Zebra.
+All incoming blocks must be validated with the protocol specifications, there is no way to avoid this logic to be present in any Zcash protocol compatible implementation such as Zebra.
 
-The amount of rules, calculations, parameters, etc needed to implement the whole Block Subsidy can be overwhelming as each little piece comes together to make a whole. Zebra is in a position where we know all the rules up to the second halving so the implementation can and will be different from `zcashd` but at the end of the day all coinbase transactions will need to be validated as the protocol describes.
-
-This document motivation is to have a clear roadmap about what is needed, the big picture of all components and how they interact together so we can separate a very big task into smaller pieces that we believe will make the implementation easier and better.
+Block subsidy and miner fees are part of the protocol, the semantic verification of them apply to the coinbase transaction and they must be verified in Zebra.
 
 # Definitions
 [definitions]: #definitions
 
 - **founders reward**: The portion of the block reward that goes into a pre defined founder address. 
 - **funding streams**:  The portion of the block reward that goes into a pre defined funding stream address.
 - **miner subsidy**: The portion of the block reward that goes into the miner of the block. 
+- **block subsidy**: Miner subsidy plus founders reward or funding stream if any of the 2 are active.
 - **coinbase transaction**: The first transaction in a block where block subsidy is done.
-- **NU**: Network Upgrade.
+- **network upgrade**: An intentional consensus rule change undertaken by the community in order to improve the network.
+- **miner fees**: The [transparent value pool](#transparent-value-pool-calculation) for all the transactions in a block.
+
+### Transparent value pool calculation
+
+There is a "transparent value pool" inside each transaction, containing "funds in motion" (distinct from the actual transparent value pool of UTXOs, being "funds at rest"). Balance of that pool works as follows:
+
+- `tx.vin` adds to the pool.
+- `tx.vout` subtracts from the pool.
+- `vpub_new` values from `tx.vJoinSplit` add to the pool.
+- `vpub_old` values from `tx.vJoinSplit` subtract from the pool.
+- `valueBalance` adds to the pool. It itself is equal to `sum(SaplingSpends) - sum(SaplingOutputs)`, so it has the same functional effect as for transparent and Sprout. 
```

Comment:
Let's split this list into Sprout, Sapling, and Transparent.

I think it will be easier to understand that way. (And let's link to the spec section that describes the `tx` fields.)